slices/ca-certificates.yaml

```diff
@@ -15,6 +23,25 @@ slices:
         content.read(certs_dir + path) for path in content.list(certs_dir)
       ]
       content.write("/etc/ssl/certs/ca-certificates.crt", "".join(certs))
+      names = [
+        "mozilla/{}".format(name) for name in content.list(certs_dir)
+      ]
+      content.write("/etc/ca-certificates.conf", "\n".join(names))
+
+  data-keep-certs:
+    essential:
+      - ca-certificates_data
+    contents:
+      /usr/share/ca-certificates/mozilla/*:
+
+  bins:
+    essential:
+      - openssl_bins
+    contents:
+      # To run update-ca-certificates without breaking the default certificates
+      # at /etc/ssl/certs/ca-certificates.crt, the ca-certificates_default-certs slice
```

Comment:
where is this `default-certs` slice?

Reply:
i think he meant the slice name, which didn't exist anymore. this is now fixed